httpx/client.py

```diff
@@ -189,8 +189,8 @@ def cookies(self) -> Cookies:
         return self._cookies
 
     @cookies.setter
-    def cookies(self, cookies: CookieTypes) -> None:
-        self._cookies = Cookies(cookies)
+    def cookies(self, cookies: Cookies) -> None:
+        self._cookies = cookies
```

Comment:
Would we want to add an `assert` or any kind of runtime type check here? I'm not sure it's really needed. Also this change makes the `cookies` property equivalent to a simple variable, so we might as well switch it to just that — a plain and public `.cookies` variable?